.circleci/config.yml

```diff
@@ -21,6 +21,7 @@ references:
           PGUSER: postgres
           PGHOST: 127.0.0.1
           POSTGRES_DB: peoplefinder_test
+          CC_TEST_REPORTER_ID: e90f692c5ece4c61d9d041c8448c0ac1c87ea6ebe72c87c3fe9aed822364ae6a
```

Comment:
Just wondering if it's worth putting this in a Circle CI env var - probably nothing sensitive right but just in case? Question really.

Reply:
@digitalali-moj ,  the id is used for uploading test coverage report (a json object containing the test coverage for each ruby file, like xxxx.rb, 49%) into https://codeclimate.com/github/ministryofjustice/correspondence_tool_staff,  so definitely no sensitive data, but as you said, I do feel a little uncomfortable to  make a "id" in plain text,  

I think it is good idea to put it in circle ci env,  could you tell me where I can set up this env? thank you